R/utils.R

```diff
@@ -19,3 +19,13 @@ check_color <- function(clr) {
   }
   return(clr)
 }
+
+check4pkg <- function(pkg) {
```

Comment:
@mnr please undo this change (i.e., put check4pkg back into zzz.R) - although perhaps these helper fxns should be in the same file, the change is completely unrelated to the focus of your PR

Reply:
This was an early change in the PR. It will be easier to reset this entire pull request and restart

Reply:
okay, sounds good

Reply:
I'll stand by on this until you have a chance to merge/reject the typo. Then I'll pull down a clean copy and rebuild the try-catch.